db-commons/src/main/resources/reference.conf

```diff
@@ -156,6 +156,13 @@ bitcoin-s {
 
 akka {
 
+  # Let Cli commands take 30 seconds
+  http.server.request-timeout = 30s
+
+  # Can parse 20MBs
+  http.server.parsing.max-content-length = 20m
+  http.client.parsing.max-content-length = 20m
```

Comment:
ditto above conf comments

Reply:
Maybe we should file an issue to disable text box for numeric outcome DLCs

Reply:
We can't know when opening the text box what kind of dlc it is :/